templates/scenarios/js/sso-tab/src/components/sample/AzureFunctions.jsx

```diff
@@ -1,23 +1,23 @@
 import { useContext } from "react";
 import { Button, Loader } from "@fluentui/react-northstar";
 import { useData } from "@microsoft/teamsfx-react";
-import { BearerTokenAuthProvider, createApiClient } from "@microsoft/teamsfx"
+import { BearerTokenAuthProvider, createApiClient } from "@microsoft/teamsfx";
 import { TeamsFxContext } from "../Context";
 import config from "./lib/config";
 
 const functionName = config.apiName || "myFunc";
 
-async function callFunction(teamsfx) {
-  if (!teamsfx) {
+async function callFunction(credential) {
```

Comment:
How about using naming convention `teamsUserCredential` to keep consistency?

Reply:
Sure, thanks

Reply:
fixed